resources/kcp/charts/kyma-environment-broker/templates/oathkeeper-rule.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: oathkeeper.ory.sh/v1alpha1
+kind: Rule
+metadata:
```

Comment:
Missing finalizer:
```
  finalizers:
  - finalizer.oathkeeper.ory.sh
```

Reply:
Isn't it automatically applied?

Reply:
Hm, yea, you are right, oathkeeper-maeaster should apply these automatically if missing.